src/engraving/libmscore/beam.cpp

```diff
@@ -604,7 +604,9 @@ int Beam::getMaxSlope() const
     // maxSlopes[i] = max slope of beam for notes with interval i
 
     // calculate max slope based on distance between first and last chords
-    double beamWidth = _elements[_elements.size() - 1]->stemPos().x() - _elements[0]->stemPos().x();
+    double endX = chordBeamAnchorX(_elements[_elements.size() - 1], ChordBeamAnchorType::Start);
+    double startX = chordBeamAnchorX(_elements[0], ChordBeamAnchorType::End);
```

Comment:
Why are we using `ChordBeamAnchorType::Start` for `endX`, and `ChordBeamAnchorType::End` for `startX`?